contrib/aws/gen-json.py

```diff
@@ -87,12 +87,13 @@
 # are started
 data['coreos']['units'] = new_units + data['coreos']['units']
 
-# configure etcd to use its EBS volume
-data['coreos']['etcd']['data-dir'] = '/media/etcd'
-
 header = ["#cloud-config", "---"]
 dump = yaml.dump(data, default_flow_style=False)
 
+# configure etcd to use its EBS volume
+dump = dump.replace('-data-dir /var/lib/etcd2', '-data-dir /media/etcd')
```

Comment:
> can't we use the old code for this?

no because the values this is inside the ExecStart section of the service. In the actual version is possible because is possible to get using the parsed yaml file